js/background.js

```diff
@@ -1428,6 +1428,9 @@
   async function connect(firstRun) {
     window.log.info('connect');
 
+    // Initialize paths for onion requests
+    await window.lokiSnodeAPI.buildNewOnionPaths();
```

Comment:
Not an issue but i would like to see what effect this has on startup time on mainnet.

Reply:
600 ms on testnet